tensorflow_datasets/core/custom_dataset/image_folder_test.py

```diff
@@ -20,30 +20,36 @@
 from __future__ import division
 from __future__ import print_function
 
-import functools
-from tensorflow_datasets import testing
-from tensorflow_datasets.core import registered
-from tensorflow_datasets.image_classification import image_folder
+import os
 
+import tensorflow_datasets.public_api as tfds
 
-class ImageLabelFolderTest(testing.DatasetBuilderTestCase):
+original_init = tfds.ImageLabelFolder.__init__
+
+def new_init(self, root_dir=None, **kwargs):
+  assert root_dir is None
+  del kwargs
+  root_dir = os.path.join(
+      tfds.core.utils.tfds_dir(), 'testing',
+      'test_data', 'fake_examples', 'image_label_folder')
+  original_init(self, root_dir=root_dir)
+
+tfds.ImageLabelFolder.__init__ = new_init
```

Comment:
Logic like this should be moved in `setUpClass(cls)` inside `ImageLabelFolderTest ` (and make sure to call `super().setUpClass()`)